Justfile

```diff
@@ -620,6 +628,17 @@ clean-pr-to-testrepo new_repo_name="test-actions-repo":
 @_foo:
     echo "example"
 
+# Developer setup: ensure environment is ready
+setup:
```

Comment:
If this is the setup of the Docker container, we should use a very different name. Setup should be something that's completely generic. If this is set up for a specific purpose, please use a more specific name and please make that clear in the above comment and in the task execution. Please clarify if this is a command that should only be run by a script and never a human. Please make this a hidden command, similar to this_ foo above.